Simplified/AxisService/NYPLAxisDRMAuthorizer.swift

```diff
@@ -9,14 +9,23 @@
 import Foundation
 
 #if AXIS
+
+@objc protocol NYPLDeviceInfoProviding {
+  var deviceID: String { get }
+  var clientIP: String { get }
+}
+
 @objcMembers
-class NYPLAxisDRMAuthorizer: NSObject, NYPLDRMAuthorizing {
+class NYPLAxisDRMAuthorizer: NSObject, NYPLDRMAuthorizing, NYPLDeviceInfoProviding {
   
-  static let deviceIDKey = "NYPLAxisDRMAuthorizerKey"
+  static private let deviceIDKey = "NYPLAxisDRMAuthorizerKey"
   
   static let sharedInstance = NYPLAxisDRMAuthorizer()
   
-  static var deviceID: String {
+  // MARK: - NYPLDeviceInfoProviding
+  let clientIP = "192.168.1.254"
```

Comment:
this hardcoded IP should probably be left out ?

Reply:
Device IP address is only needed when generating a url for downloading license. Android is also hardcoding the ip address. Looking at it from a Cost vs Benefit perspective, I'm under the impression that generating device IP address would be more costly and would provide a result no different than what the hard coded IP address already provides. 

However I'm open to suggestions and will be more than happy to not having to hardcode it if you think we should generate the ip address for the device.

Reply:
I see.... so this IP can be any number? If so can you add some documentation explaining why we need this and that it doesn't actually have to be the real client IP
 (assuming I understood correctly)?  Otherwise developers reading this code won't know why we have this constant here.

Reply:
Agreed. Done.